### PR TITLE
Dashboard command revamp

### PR DESCRIPTION
**Pull request by erenJag**

### CODE REVIEW

**Review by registergoofy — APPROVED** [review]
LGTM

File: cmd/crowdsec-cli/dashboard.go [hunk 1]
```diff
@@ -106,20 +113,42 @@ cscli dashboard setup -l 0.0.0.0 -p 443
...
 				log.Fatalf("Failed to stop metabase container : %s", err)
 			}
 		},
 	}
-	cmdDashStop.Flags().BoolVarP(&remove, "remove", "r", false, "remove (docker rm) container as well.")
 	cmdDashboard.AddCommand(cmdDashStop)
+
+	var cmdDashRemove = &cobra.Command{
+		Use:   "remove",
+		Short: "removes the metabase container.",
```
Comment (buixor) [inline cmd/crowdsec-cli/dashboard.go:131]: I think we need a "purge" option that is going to remove docker image to tidy up our mess :)

**Review by buixor — CHANGES_REQUESTED** [review]
minor change needed for the removal, but lgtm

File: cmd/crowdsec-cli/dashboard.go (hunk 1 above)
Comment (erenJag) [reply cmd/crowdsec-cli/dashboard.go:128]: fixed by adding the feature in the `remove --force`

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**